libusb/libusb.h

```diff
@@ -2279,3 +2279,11 @@ int LIBUSB_CALLV libusb_set_option(libusb_context *ctx, enum libusb_option optio
 #endif
 
 #endif
+
+/** \ingroup libusb_desc
+ *  Retrieve the name of the backend driver.
+ *
```

Comment:
Should have a line analogous to "Since version 1.0.23, \ref LIBUSB_API_VERSION >= 0x01000107", see elsewhere.

Reply:
I assume this will be part of the next release so I used:

```
Since version 1.0.27, \ref LIBUSB_API_VERSION >= 0x0100010B
```